sys/evtimer/evtimer.c

```diff
@@ -153,6 +153,26 @@ void evtimer_del(evtimer_t *evtimer, evtimer_event_t *event)
     irq_restore(state);
 }
 
+uint32_t evtimer_get_offset(evtimer_t *evtimer, evtimer_event_t *event)
+{
+    uint32_t offset = 0;
+    unsigned state = irq_disable();
+    if (evtimer->events) {
+        evtimer_event_t *list = evtimer->events;
+        /* Get current offset to first event */
+        offset = _get_offset(&evtimer->timer);
+        while (list->next) {
```

Comment:
how about ```while ((list = list->next))```? I've used that style in the rest of this file.

Reply:
Fixed